datarepo-clienttests/src/main/java/utils/AssertUtils.java

```diff
@@ -0,0 +1,17 @@
+package utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AssertUtils {
```

Comment:
This is fine for now. It would be worth exploring whether we can use the hamcrest matchers, instead of having to write each match ourselves. Then we could just write a variation of assertThat.

Reply:
Oh, that's much better. I went ahead and switched to assertThat/equalTo.